tests/snapshot/README.md

```diff
@@ -0,0 +1,47 @@
+# Snapshot testing
+
+Nickel snapshot tests are run by invoking the most recently built Nickel binary
+(located via the `CARGO_BIN_EXE_nickel` environment variable) with the Nickel
```

Comment:
Is `nickel` in lowercase on purpose, in `CARGO_BIN_EXE_nickel`? Or is it a typo?

Reply:
It's deliberate - if you look at `CARGO_BIN_EXE_<name>` in [the docs](https://doc.rust-lang.org/cargo/reference/environment-variables.html) it says:

> The <name> is the name of the binary target, exactly as-is. For example, CARGO_BIN_EXE_my-program for a binary named my-program.